### DISCUSSION TITLE
To be and not to be Philosophical

### DISCUSSION THREAD

**Discussion by khuramonline** [a-cosmology-group/acg#90] [Metaphysics and Philosophy]
### Non-Philosophical Thinking

Non Philosophical thinking is disjointed and non-comprehensive. Mainly the focus is only on particular issues or problems. Overall it is non-critical and tends to accept point of view of the authoritative sources. This type of thinking takes various points of views as settled or for granted facts therefore no need is felt for the personal verification etc.

### To be Philosophical

To be Philosophical means looking things in totality. To have a totalitarian approach. Rather than focusing a particular problem, trying to understand overall general structure, scheme or theme. To have a systematic approach and unwillingness to accept any stance within that general scheme without having solid logical justification.

This philosophical approach is good because non Philosophical people are often trapped in particular problems that may not even have sound logical basis. Having totalitarian approach is not a hindrance for solving particular problems either. Philosophy is more interested in big picture. A big picture philosopher can be dealing with particular problems but that would be his practical activity rather than philosophical.

### Academic Philosophy

Academic Philosophy is an artificial form of non-critical philosophy. It is just Academic Philosophy and not Philosophy.

**Comment by OlliSant** [comment]
Yes, it was wrong to leave the cosmology to physicists, because its problems are philosophical problems. We need both philosophy and astronomy, in this order. Of course a good astronomer understands the philosophical problems, but the astronomers tend to be empiricists, when in philosophy we need good balance between rational thinking and study and empirical study.

 Holistic and dialectical are better terms as totalitarian.

Philosophy of cosmology 
Theology of cosmology
Sociology of cosmology
Psychology of cosmology
Biology of cosmology
Physics of cosmology

All that is needed in science and philosophy of cosmology.

**Reply by budrap00** [reply]
Science is the study of physical reality and as such it is constrained by empiricism to studying those things that can be directly observed, detected, and measured. Philosophy is the study of human ideas. They are not the same thing. To the extent that science was once a branch of philosophy (natural philosophy) it separated itself by its empirical restriction. Since that separation science has been wildly successful in a way that philosophy never has been. Science can launch rockets to the Moon, the only thing philosophy has been known to successfully launch is an argument.

This is not a trivial matter. The empirical constraint is what has made science successful - by making it something other than philosophy. Philosophy does not have an empirical constraint; its subject matter is the boundless realm of the human imagination. The unconstrained human imagination has never been a reliable guide to the nature of physical reality.

I'm reading a book called Quantum Reality by Jim Baggott. You could say that Baggott is a philosopher of science and as such he has interesting thoughts on the quantum realm - because he has an understanding of science as an empirical discipline. Philosophical ideas couched in that empirical understanding of science are especially interesting in the case of the quantum realm because therein lies an apparent hard empirical boundary. 

The inability to directly detect an electron in a double slit experiment except at its origin and detection points appears not to be merely a technological limitation. It appears to be a hard limit imposed by the scale of the electron which renders it susceptible to disruption by any known means of direct detection. The question of how science is to consider the, let's call it near field, just beyond it's hard limits is an interesting one that obviously runs along the border between empirically bound science and unbound philosophy. Something of a physical nature has to transpire between emission and absorption of the electron. In those types of discussions a scientifically literate philosopher like Jim Baggott may well have interesting and relevant thoughts. 

But the fact remains that philosophy and mathematics and theology are not scientific disciplines because they lack an empirical constraint and lacking that constraint, any thoughts they might offer on scientific endeavors has to be considered unreliable unless they are clearly informed or supported by empirical evidence. It is not the job of science to chase after philosophical, mathematical or theological speculations that lack empirical, scientific motivation, but are instead motivated only by some philosophical, mathematical, or theological predilection.

**Reply by khuramonline** [reply]
Yes Science should be constrained by Empiricism. But in practice, Theoretical Physics has been detached from Empiricism. Practical and ongoing [Theoretical Physics is under the trap of Idealism](https://github.com/orgs/a-cosmology-group/discussions/54) and it is clear that the [methodology of Theoretical Physics is same as Rationalism of René Descartes](https://github.com/orgs/a-cosmology-group/discussions/79).

Moreover, Empiricism itself is a form of Philosophy. Term 'philosophy' does not necessarily mean that its method has to be other than Empiricism.

The title of my first book is "A Philosophical Rejection of the Big Bang Theory". Following is written in the book description of amazon page:

> Scientific inquiry takes onward course from the point where previous scientists had reached. But philosophical analysis initiates from scratch. Philosophy questions everything and chooses starting point for itself after having ruled out all the unsubstantiated and doubtful elements of the topic under study. Secondly, known realities must make sense. If a theory is officially 'counterintuitive', then either it is mere fiction or at the most; a distorted form of truth. **_This book's analysis is based on the philosophical principle that knowledge is empirical and does not arise magically in absence of observational grounds._** With philosophical approach, it was doubtful to accept that Georges Lemaître already knew Hubble's law in year 1927 that was yet to be found by Edwin Hubble in year 1929.

So in this case, philosophy was not detached from Empiricism. It was the "Theoretical Physics" that was detached from Empiricism.

And I sufficiently showed that **the claim** that based on mathematical derivations, Georges Lemaître already knew Hubble's law in year 1927 that was yet to be found by Edwin Hubble in year 1929 **[is not a correct claim](https://github.com/orgs/a-cosmology-group/discussions/36)**.

Therefore "Knowledge is Empirical" is not a non-philosophical assertion.  Knowledge comes from Empirical facts as supported by Empirical Analysis **and not from mere mathematical derivations**.

**Comment by OlliSant** [comment]
This is empirical definition of science. Philosophy is the broader view of things what we think are scientific. Sciences are a part of philosophy, always, and philosophy is a still broader thing, consisting the world views and religions and the skills of the life. And ethics, politics and aesthetics. And science is not only the natural sciences but also psychology, history and the social sciences and theology. In English the science means the natural sciences, but this is not all what science is. 

In Western philosophy the science is separated of other things, but in India philosophy, sciences, religions and all things are an united whole. Both have their merits. The point is to achieve the truth of everything and of all parts of everything.
